include/openthread-coap.h

```diff
@@ -224,6 +239,31 @@ ThreadError otCoapHeaderAppendOption(otCoapHeader *aHeader, const otCoapOption *
 ThreadError otCoapHeaderAppendUriPathOptions(otCoapHeader *aHeader, const char *aUriPath);
 
 /**
+ * This function appends a Max-Age option.
+ *
+ * @param[inout]  aHeader   A pointer to the CoAP header.
+ * @param[in]     aMaxAge   The Max-Age value.
+ *
+ * @retval kThreadError_None         Successfully appended the option.
+ * @retval kThreadError_InvalidArgs  The option type is not equal or greater than the last option type.
+ * @retval kThreadError_NoBufs       The option length exceeds the buffer size.
+ */
+ThreadError otCoapHeaderAppendMaxAgeOption(otCoapHeader *aHeader, uint32_t aMaxAge);
+
+/**
+ * This function appends a single Uri-Query option.
+ *
+ * @param[inout]  aHeader   A pointer to the CoAP header.
+ * @param[in]     aUriQuery A pointer to NULL-terminated string, which should contain
+ *  a single key=value pair.
```

Comment:
By alignment I meant to align it to a column where description begins (some other docs do it this way):
```
* @param[in]     aUriQuery A pointer to NULL-terminated string, which should contain
*                          a single key=value pair.
```
But a single line should work as well 👍